Makefile

```diff
@@ -156,13 +156,14 @@ devnet-fork:
 	@echo "--> Running devnet fork script..."
 	@python3 contrib/devnet/devnet_fork.py --node-version $(OLD_VERSION:v%=%)
 
+DEVNET_UPGRADE_VERSION := v37.0.0
 devnet-fork-upgrade:
 	@echo "--> Running devnet fork script with upgrade..."
 	@python3 contrib/devnet/devnet_fork.py --node-version $(OLD_VERSION:v%=%) --upgrade-version $(DEVNET_UPGRADE_VERSION)
 
 download-snapshot:
-	@echo "--> Downloading and caching testnet snapshot..."
-	@python3 contrib/devnet/download_snapshot.py
+	@echo "--> Downloading and caching snapshot..."
+	@python3 contrib/localnet/scripts_python/download_snapshot.py --chain-id $(or $(CHAIN_ID),athens_7001-1)
```

Comment:
Do you think it's good idea to have both `download-mainnet-snapshot` and `download-testnet-snapshot` with separate hardcoded `chain-id`?

Reply:
I wanted to keep 1 script to be able to resuse as much code as possible